pkgs/development/python-modules/pagelabels/default.nix

```
@@ -0,0 +1,19 @@
+{ stdenv, buildPythonPackage, fetchPypi, pdfrw }:
```

add a line please :)
```suggestion
{ stdenv, buildPythonPackage, fetchPypi, pdfrw }:

```

---

just curious, do you refer to a specific guideline to suggest the added line, or is that sthg you have seen in a formatter ?